packages/airnode-examples/integrations/config-utils.ts

```diff
@@ -0,0 +1,55 @@
+import { writeFileSync } from 'fs';
+import { join } from 'path';
+import { Config, LocalOrCloudProvider } from '@api3/airnode-node';
+import { cliPrint, getDeployedContract, readChainId, readIntegrationInfo } from '../src';
+
+export const createCloudProviderConfiguration = (generateExampleFile: boolean): LocalOrCloudProvider => {
+  if (generateExampleFile) {
+    return {
+      type: 'local',
+    };
+  }
+
+  const integrationInfo = readIntegrationInfo();
+  const airnodeType = integrationInfo.airnodeType;
+
+  switch (airnodeType) {
+    case 'aws':
+      return {
+        type: airnodeType,
+        region: 'us-east-1',
+      };
+    case 'local':
+      return {
+        type: airnodeType,
+      };
+    case 'gcp': {
+      return {
+        type: airnodeType,
+        region: 'us-east1',
+        projectId: integrationInfo.gcpProjectId!,
+      };
+    }
+  }
+};
+
+export const getAirnodeRrpAddress = async (generateExampleFile: boolean) => {
+  if (generateExampleFile) return '0x5FbDB2315678afecb367f032d93F642f64180aa3';
+
+  const airnodeRrp = await getDeployedContract('@api3/airnode-protocol/contracts/rrp/AirnodeRrp.sol');
+  return airnodeRrp.address;
+};
+
+export const getChainId = async (generateExampleFile: boolean) =>
+  (generateExampleFile ? 31337 : await readChainId()).toString();
+
+export const createNodeVersion = () => {
+  return '0.2.2';
```

Comment:
Will this be read from the `package.json` or what's the plan here? Because as soon as 0.3 is out this will break, won't it?

Reply:
It will need to be manually updated for now, before the release. But it's only this one number that needs to be modified.